lib/wallet/walletdb.js

```diff
@@ -2279,6 +2279,8 @@ class WalletDB extends EventEmitter {
         tip.hash, total);
     }
 
+    this.emit('block connect', entry, txs);
```

Comment:
Question for you and other reviewers: Should we filter the `txs` array emitted here so only TXs added to the wallet are emitted? We could use the logic on line 2267 above and return a new array of only TXs that belong to us... unless there's a reason why we would need the entire block?

Reply:
Updated to filter only wallet tx. Note that it's not for a specific wallet; the tx will count if it belongs to at least 1 wallet.